config/crd/bases/infrastructure.cluster.x-k8s.io_proxmoxclusters.yaml

```diff
@@ -91,6 +91,19 @@ spec:
                     items:
                       type: string
                     type: array
+                  allocateReservedIPAddresses:
```

Comment:
I don't think this field should ever be true for our use case. This might be a bit out of scope, but we should have a custom struct for the IPAM spec. This work was already started in another PR, but it should not depend on it.

Reply:
Oh, yeah 
I agree

Reply:
I have removed the ipam dependency, 
that will make it easier to maintain.